### ISSUE TITLE
logits_cw and logits_r are nan during training

### ISSUE THREAD

**Issue by Kevinddddddd** [zcemycl/TF2DeepFloorplan#32]
Hi @zcemycl,

Great work! 

I run your code on RTX3070 with tensorflow 2.2-gpu, and the outputs of logits_cw and logits_r are nan during training. But I run your code again on RTX2080Ti with tensorflow 2.3-gpu, that situation does not happen. I don't know why.

I'd like to know how to test the accuracy based on your rewrote codes. Did you rewrite this part? 

Thanks very much!

**Comment by zcemycl** [comment]
@Kevinddddddd I would suspect tf2 works differently with RTX3070, make sure if you install tf2 correctly regarding to your RTX3070. I have no RTX3070 so I can't really know. 

Can you try finding out which tf function is causing it? Maybe some logorithm functions? Or backprops? 

About the accuracy, i would suggest you to try 1-(L2 norm)/(total number of pixels)? So if L2 norm is 0, the accuracy will be 100%. Or cross entropy function since you can get multiclass classifications after postprocess (not colorise)? Or class dependent F1 score?

**Comment by Kevinddddddd** [comment]
Okay, thanks for your suggestions.

Best regards,
Kevin
